doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,939 @@
+  * Documentation for Asset tracker v2 :ref:`asset_tracker_unit_test`.
+  * Support for Lightweight Machine to Machine (LwM2M).
+  * Support for filtering updates to cloud based on LTE connection evaluation (`AT%CONEVAL`_).
+
+* Updated:
+
+  * For nRF Cloud builds, the configuration section in the shadow is now initialized during the cloud connection process.
+  * Allow the :ref:`lib_nrf_cloud` library to handle modem FOTA updates if :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` is enabled.
+
+nRF9160: Serial LTE modem
+-------------------------
+
+* Added:
+
+  * ``#XDFUGET`` and ``#XDFURUN`` AT commands to support the cloud-to-nRF52 DFU service.
+  * Native TLS support to the HTTPS client.
+  * ``#XCMNG`` command to support the use of native TLS.
+  * ``#XSOCKETSELECT`` AT command to support multiple sockets in the Socket service.
+  * ``#XPOLL`` AT command to poll selected or all sockets for incoming data.
+
+* Updated:
+
+  * Enhanced the ``#XHTTPCREQ`` AT command for better HTTP upload and download support.
+  * Enhanced the ``#XSLEEP`` AT command to support data indication when idle.
+  * Enhanced the MQTT client to support the reception of large PUBLISH payloads.
+
+* Fixed:
+
+  * The secondary MCUboot partition information is no longer passed to the P-GPS library if the P-GPS partition is enabled.
+  * The combined use of A-GPS and P-GPS so that ephemeris and almanac data is not requested via A-GPS, saving both power and bandwidth.
+
+nRF Machine Learning (Edge Impulse)
+-----------------------------------
+
+* Increased the value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds for Thingy:53.
+
+nRF Desktop
+-----------
+
+* Added documentation for selective HID report subscription in :ref:`nrf_desktop_usb_state` using the :ref:`CONFIG_DESKTOP_USB_SELECTIVE_REPORT_SUBSCRIPTION <config_desktop_app_options>` option.
+
+* Removed configurations without a bootloader.
+  The B0 bootloader is enabled by default on all boards if the configuration with two image slots fits in memory.
+  Alternatively, MCUboot bootloader with a single image slot and serial recovery is enabled.
+  In case the configuration with MCUboot does not fit in memory, no bootloader is enabled.
+
+Thingy:53 Zigbee weather station
+--------------------------------
+
+* Added new :ref:`zigbee_weather_station_app` application.
+* Fixed an issue where the buffer was being freed incorrectly.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Bluetooth samples
+-----------------
+
+* Added the :ref:`peripheral_ams_client` sample.
+
+
+* Updated:
+
+  * :ref:`direct_test_mode` sample:
+
+    * Updated:
+
+      * The front-end module test parameters are not set to their default value after the DTM reset command anymore.
```

Comment:
Please remove this line as this is a duplicate of line 253

Reply:
done